### ISSUE TITLE
[Bug]: 5.2.0 on Linux, app indicator always flashes between Linphone icon and the number "99"

### ISSUE THREAD

**Issue by xenotropic** [BelledonneCommunications/linphone-desktop#816]
### Context

When I launch the application, the notifier icon in the tray continually alternates between the Linphone logo and the number 99, about once per second. The blinking is distracting, and also it makes no sense. 99 . . . notifications? But clicking everywhere I can't find _any_ messages or notifications. It's connected to Voip.ms so theoretically it makes sense that there would maybe be some notifications to give. Pretty sure this did not happen on 5.0.2.

### General information

- Device: Lenovo ThinkPad X1 carbon 3rd
- OS: Ubuntu 
- Version of the App: Desktop 5.2.0 - Qt5.15.10
Core 5.3.1 (from OpenSuse, as recommended to me in issue 811)
- Version of the SDK: ?

### Expected behaviour

Just show the logo, no blinking number. 

### To Reproduce

I just launched the application; it is connected to a voip.ms account that does have an existing call and sms message history, but can't see either one in Linphone.

**Comment by xenotropic** [comment]
This persists with 5.2.1.

I'm adding illustrations, it alternates between these every second

![image](https://github.com/BelledonneCommunications/linphone-desktop/assets/4050617/6dfdadfe-6680-473f-887f-53e56e9a6119)

![image](https://github.com/BelledonneCommunications/linphone-desktop/assets/4050617/23db3635-0e5f-4962-982b-def770bdaf96)

I've searched the application in vain for any "clear notifications" facility. I don't even know what the notifications could be. I deleted all my call history, so that's not it. Although I do receive SMS messages with this Voip.ms account, when I click the chat bubble on the left panel, there's no messages there so it seems like Linphone is not receiving them for whatever reason (fine by me), so seems like those can't be causing the 99 either.
